ct.sh

```diff
@@ -10,7 +10,7 @@ if [ -z $TRAVIS_RUST_VERSION ]; then
 fi
 
 if [ "$TARGET" == "aarch64-unknown-linux-musl" ]; then
-  pushd /tmp && wget https://musl.cc/aarch64-linux-musl-cross.tgz;
```

Comment:
Verify known checksum while you're at it?

Reply:
Ah good point!